packages/flame_jenny/jenny/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: jenny
-description: Yarn Spinner equivalent for Dart.
-version: 0.0.0
+description: YarnSpinner equivalent for Dart.
+version: 1.0.0
```

Comment:
This is done by Melos

Reply:
how would it know that the version should go to 1.0.0 and not to, say, 0.0.1?

Reply:
You can choose it for the initial version, or for any version really if you want to break semver which we do constantly. 😅